app/converters/shopify/part_converter.rb

```diff
@@ -0,0 +1,31 @@
+module Shopify
+  class PartConverter
+    def initialize(part:, permutation:)
+      @part = part
+      @permutation = permutation
+    end
+
+    def to_hash
```

Comment:
Yeah I think you're right. It doesn't make much sense to have it as being a class. But at the same time I thought of ActiveRecord that converts a regular object `.to_hash` without any parameters and converts the object to an hash. 

I see your point but I am unsure if this would "confuse" people that you have to pass params to the `to_hash` method, but at the same time, it's confusing having to initialize it. IDK :)